www/scratch-website.splash-l10njson/mi.json

```diff
@@ -11,7 +11,7 @@
     "splash.communityLoving": "Ko ngā Mea e Arohaina ana e te Hapori",
     "messages.becomeCuratorText": "I eke a {username} hei kairauhī mō {studio}",
     "messages.becomeManagerText": "I tokona ake a {username} hei kaiwhakahaere mō {studio}",
-    "messages.favoriteText": "I tohua e {profileLink} he pai te {profileLink}",
+    "messages.favoriteText": "Nā {profileLink} i kaingākau ki tō kaupapa {projectLink}",
```

Comment:
This translation, with the corrected placeholder, has been waiting in Transifex for a while. I copied it into this file to make the tests pass.